tests/TestBackendSVG.cpp

```diff
@@ -69,9 +69,12 @@ TEST_CASE("figure can written using SVG backend", "[figure]") {
     do_plot(i);
   }
 
+  ax->font_face("Indie Flower");
+
   std::ofstream out;
   out.open("test_figure.svg");
   BackendSVG backend(out);
+  backend.import_web_font("https://fonts.googleapis.com/css?family=Indie+Flower");
```

Comment:
Be nice if this could get added automatically. I.e. if a google web font is chosen via the `ax->font_face` call then this will call `backend.import_web_font`. Would this work?

Reply:
Hmm.  My feeling is the two ought to be kept apart.  If you have `Indie Flower` installed on your machine then you might want to omit the `import_web_font` call, and I don't think we want to be trying to identify whether fonts are installed.

Conceivably you could accept a URL to the `font_face()` method on `Axis`, but then you would need logic to determine if a URL or a name were being passed, and I assume this would be a problem WRT backends for which web fonts don't make sense.

Thoughts?

Reply:
I guess my main point would be, given there are to be multiple backends I think the frontend stuff shouldn't know about anything backend specific.

Would web fonts make any sense in the GL backend?

Reply:
On the otherhand, there are probably too many google web fonts to do that. Perhaps the perhaps the `ax->font_face` has an optional bool argument that is set to `true` if the user wants a web font?

Reply:
Whoops, didn't see your last two comments. Yes, you are right, the frontend shouldn't know about any backend specific stuff. Mind you, the gl backend could probably just download a web font and start using it right? In any case, I'm happy to merge what is there now, we can keep playing with it later if necessary.